cfpb/consumerfinance.gov
# Notifications: Consolidate type settings | remove unused default class | change 'default' to 'information' behind the scenes

`m-notification__default` added in https://github.com/cfpb/consumerfinance.gov/pull/5742 is an unused modifier and can be removed.

## Removals

- `m-notification__default` modifier.


## Changes

- Consolidates notification type and icon settings together to make it clearer what the combonations are.
- Changes `default` notification to `information`, since that's how it is used.
- Minor whitespace cleanup.


## How to test this PR

1. Check out branch and run migrations
2. Create a new Browse or Learn page, add a notification, and verify that default Information style can be applied to it
3. Clicking through http://localhost:8000/paying-for-college/your-financial-path-to-graduation should show unchanged notification appearances.

## Notes and todos

- Design system update for default notification is here https://github.com/cfpb/design-system/pull/1212
